ui/packages/catalog-ui-search/src/main/webapp/component/query-template-sharing/query-template-sharing.less

```diff
@@ -0,0 +1,111 @@
+@import '../editable-rows/editable-rows';
+
+.icon-label {
+padding-left: 8px;
+}
+
+@{customElementNamespace}template-sharing {
+
+  @valueWidth: 60%;
+  position: relative;
+
+  .customElement;
+
+  .text-right {
+    text-align: right;
+  }
+
+  overflow: auto;
+
+  .sharing-buttons {
+    position: absolute;
+    bottom: 20px;
```

Comment:
Should we use @minimumSpacing here and the next line?

Reply:
Setting these to minimum spacing goes against what workspaces has and alters the alignment to be misaligned in this case. I'm open to suggestions or it might make sense to leave this as it is.